### ISSUE TITLE
Releases without the package

### ISSUE THREAD

**Issue by ZaheenJ** [AndydeCleyre/archbuilder_iosevka#1]
So the build fails at "copy ttf-iosevka-custom-git PKGBUILD into build ctnr" with this error:
```
Run buildah copy --chown dev ctnr pkgs/ttf-iosevka-term-custom-git /home/dev/ttf-iosevka-term-custom-git
error adding content to container "ctnr": error checking on source /home/runner/work/archbuilder_iosevka/archbuilder_iosevka/pkgs/ttf-iosevka-term-custom-git under "/": copier: stat: "/home/runner/work/archbuilder_iosevka/archbuilder_iosevka/pkgs/ttf-iosevka-term-custom-git": no such file or directory
level=error msg="exit status 125"
Error: Process completed with exit code 125.
```
I followed the README's instructions for using the customizer site. Here is my vars.yml:
```
---
build:
  - "ttf-iosevka-custom-git"
```
Here is my private-build-plans.toml.wz
```
[buildPlans.iosevka-custom]
family = "Iosevka Custom"
spacing = "normal"
serifs = "sans"

[buildPlans.iosevka-custom.variants.design]
capital-d = "more-rounded"
capital-g = "toothless-rounded-serifless-hooked"
capital-i = "serifed"
capital-j = "serifless"
capital-k = "symmetric-touching"
capital-p = "closed"
capital-q = "crossing"
capital-r = "straight"
capital-w = "straight"
a = "double-storey-toothless-corner"
b = "toothless-corner"
d = "toothless-corner"
e = "flat-crossbar"
f = "serifless"
g = "earless-corner"
i = "serifless"
j = "serifless"
k = "symmetric-touching"
l = "serifless"
m = "earless-corner-double-arch"
n = "earless-rounded-straight"
p = "earless-corner"
q = "earless-corner"
r = "earless-corner"
t = "flat-hook"
u = "toothless-rounded"
w = "straight"
y = "cursive"
cyrl-capital-u = "straight"
zero = "slashed-oval"
four = "closed-non-crossing"
six = "open-contour"
nine = "open-contour"
asterisk = "hex-low"
underscore = "high"
caret = "medium"
paren = "normal"
brace = "straight"
number-sign = "slanted"
at = "fourfold"
percent = "rings-continuous-slash"
bar = "natural-slope"
question = "corner"

  [buildPlans.iosevka-custom.ligations]
    inherits = "clike"
```
Here is the full build log:
[logs_3.zip](https://github.com/AndydeCleyre/archbuilder_iosevka/files/6005288/logs_3.zip)
This method of building iosevka is amazing, not having to have the huge repo on my computer is great! I have never even heard of workflows before, great work!

**Comment by AndydeCleyre** [comment]
Firstly: Thank you very much. As I don't use the customizer site, this is a very valuable report for me.

---

Maybe my documentation can be improved, and I'll accept suggestions or PRs for that, but I think the issue here that you've skipped the "generate your new workflow" step under "How to Use" (bullet 4).

**Comment by ZaheenJ** [comment]
I thought I did but I guess I didn't. Either way that didn't work. However I tried something different. Instead of getting rid of the default settings in vars.yaml and having the one shown above, I just ran `yaml-merge -A right -w vars.yml vars.yml <<<'{"build": ["ttf-iosevka-custom-git"]}'`. Then everything worked like it should. I didn't at first because I was afraid those settings would override the private-build. Thank you for this!
